atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1195,18 +1270,45 @@ private static boolean isSweepSentinel(Value value) {
      * This will return all the keys that still need to be postFiltered.  It will output properly
```

Comment:
This isn't quite accurate (why is it a Map?) Maybe KV pairs, or return the keys ... along with their values.

Reply:
Added KV pairs description.